part_one.asm

```diff
@@ -0,0 +1,97 @@
+# COMP3410 Program Template
+# Author: Edward Prins
+# Assignment: Programming Assignment #1, Part 1
+# Due Date: 11 February 2015
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+#size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+input:	.asciiz "\Enter the size of the array\n"
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+              la   $a0, input
+              li   $v0, 4
+              syscall
+              
+              li   $v0, 5
+              syscall
+              
+              move $s5, $v0
```

Comment:
-5 Accepting input greater than declared array size is dangerous because your writing to memory that's outside the bounds of the array.